CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# v1.0.1 (2016-01-31)
+
+* Optimization
+* Add gulp build task
+* Add NPM support
+
+# v1.0.0 (2016-01-31)
```

Comment:
the date here is incorrect, otherwise looks good!

Reply:
Thanks for catching this. Fixed and merged.